docs/spec/sanitations.md

```diff
@@ -1,24 +1,42 @@
-_Author_:  <!-- TODO: Add author name --> \
-_Created_: <!-- TODO: Add date --> \
-_Updated_: <!-- TODO: Add date --> \
+_Authors_: @manodyaSenevirathne \
+_Created_: 2024/08/05 \
+_Updated_: 2024/08/05 \
 _Edition_: Swan Lake
 
 # Sanitation for OpenAPI specification
 
-This document records the sanitation done on top of the official OpenAPI specification from OpenAI Chat. 
-The OpenAPI specification is obtained from (TODO: Add source link).
+This document records the sanitation done on top of the official OpenAPI specification from OpenAI Chat.
+The OpenAPI specification is obtained from the [OpenAPI specification for the OpenAI API](https://github.com/openai/openai-openapi/blob/master/openapi.yaml). 
 These changes are done in order to improve the overall usability, and as workarounds for some known language limitations.
 
-[//]: # (TODO: Add sanitation details)
-1. 
-2. 
-3. 
+1. **Removed the `Deprecated: true` property from the below schemas**:
+
+   - **Original**:
+      - `Deprecated: true`
+
+   - **Updated**:
+      - Removed the `deprecated` parameter
+
+   - **Reasons**: The original configuration was generated successfully, but it led to compile-time warnings. To ensure smoother compilation and avoid potential issues, the configuration has been revised.
```

Comment:
Shall we mention the affected schemas with this change? (just like we've mentioned in the 2nd sanitation)

Reply:
Sure, I'll update the documentation